scenes/menus/title/components/title_screen.gd

```diff
@@ -4,21 +4,11 @@ extends Control
 
 @export var next_scene: PackedScene
 
-@onready var button_box: VBoxContainer = %ButtonBox
-@onready var start_button: Button = %StartButton
+@onready var main_menu: Control = %MainMenu
+@onready var credits: Control = %Credits
 
 
-func _ready() -> void:
-	while Pause.is_paused(Pause.System.PLAYER_INPUT):
-		await Pause.pause_changed
-
-	start_button.grab_focus()
-
-
-func _on_start_button_pressed() -> void:
-	# Prevent repeated input
-	button_box.queue_free()
```

Comment:
I removed these workarounds because I think they will be fixed by https://github.com/endlessm/threadbare/pull/384.